doc/radosgw/lua-scripting.rst

```diff
@@ -424,3 +428,17 @@ Then, do a restart for the radosgw and upload the following script to the `postR
     assert(s:close())
   end
 
+
+- Trace only for operations of specific bucket
+
+If `tracing is enabled <https://docs.ceph.com/en/latest/jaegertracing/#how-to-enable-tracing-in-ceph/>`_ on the RGW, the value of Request.Trace.Enable is true, so we should disable tracing for all other requests that do not match the bucket name.
```

Comment:
can we do the opposite as well?
what if tracing is disabled and we want to turn it on dynamically in the script?

Reply:
yes. but we need to distinguish between the `false` value that was set by Lua script and the default one.
it sounds like a better solution than the process of `enable tracing->disable irrelevant requests`
maybe we should use an `enum` for that? so the default value will be different than false?

Reply:
one a different thought, I think it isn't necessary. I'll try to do that.

Reply:
see [2ed430c](https://github.com/ceph/ceph/pull/45662/commits/2ed430c9d02890d210f9d32a4326241aacb72cb0)

Reply:
this is great. please update doc and the example